assets/js/googlesitekit/api/index.js

```diff
@@ -355,6 +355,7 @@ const API = {
 	set,
 	setUsingCache,
 	usingCache,
+	apiFetch,
```

Comment:
I don't think there's any need to export this from our API library, which is more specific to accessing our own WP REST API endpoints.

Instead, you can just import this directly in the file that needs it from the `@wordpress/api-fetch` package.

Reply:
Thanks, updated. 
It was originally suggested in IB, so I included it. But it is removed now, in favour of using it directly from the package